include/envoy/ssl/context_config.h

```diff
@@ -50,7 +50,7 @@ class ContextConfig {
   /**
    * @return CertificateValidationContextConfig the certificate validation context config.
    */
-  virtual const CertificateValidationContextConfig* certificateValidationContext() const PURE;
+  virtual CertificateValidationContextConfig* certificateValidationContext() const PURE;
```

Comment:
Why did this become non-const?

Reply:
This is because I added `Api::Api& api()` to `CertificateValidationContextConfig` which cannot be const if I understand correctly. API object is necessary for SPIFFE validator to retrieve trust bundles from DataSource. If there's a workaround, I would like to adopt. thoughts?

Reply:
https://github.com/envoyproxy/envoy/pull/14884/commits/63e4901a74abcef1910d91f410180128de4dc6e5 fixed!